moabb/paradigms/p300.py

```diff
@@ -120,7 +126,7 @@ def process_raw(self, raw, dataset):
             if self.resample is not None:
                 epochs = epochs.resample(self.resample)
             # MNE is in V, rescale to have uV
-            X.append(1e6 * epochs.get_data())
+            X.append(self.unit_factor * epochs.get_data())
```

Comment:
since this is within `process_raw` it has access to the dataset object -- I agree with your comment, this unit factor should *not* be a part of the paradigm but rather the dataset itself. We'll have to better schematize datasets so we can enforce what's in them...if I get bored and come back to this, the `attr` library is a promising candidate

Reply:
I see. I can implement it to look for a unit_factor attribute in the dataset and if it doesnt exist uses the default 1e6?

Reply:
that would be great